### PR TITLE
docs: Document the DB schema

### PR DESCRIPTION
**Pull request by jasonborg**
Add documentation for the DB schema used by the Snapshot Debugger.

This addresses #125

### CODE REVIEW

File: docs/SCHEMA.md
Comment (mctavish) [inline docs/SCHEMA.md:1]: Nit: I'd switch the order around below to be:
* schema_version
* debuggees
* breakpoints

That's the order in which the pieces should be understood in order to get a grasp of the system as a whole.

File: docs/SCHEMA.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+user. For finalized `Snapshot` breakpoints that have capture data, the full data
+is present here, meaning the `stackFrames`, `variableTable` and
+`evaluatedExpressions` are present, unlike under the `final` node.
+
+### Breakpoint Entity
+
+The breakpoint entity is fully documented in protobuf form
+[here][breakpoint_entity]. The data in the Firebase DB is in JSON format, which
+protobuf maps to as described [here][json_mapping]. One thing to note
+is how Firebase stores [arrays][firebase_arrays].
```
Comment (mctavish) [inline docs/SCHEMA.md:102]: I'd move this up to the top, before we go into the active, final, snapshot details.  Those details refer to members of the proto, so it's good to have the proto up front.

File: docs/SCHEMA.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+## cdbg/debuggees
+
+This node contains information about the debuggees themselves. See the section
```
Comment (mctavish) [inline docs/SCHEMA.md:106]: Probably a couple sentences about how the agents use this field.

A consistent ID is selected by the agent, the debuggee details are written on agent startup, and the lastUpdateTimeUnixMsec is periodically updated to track active debuggees.

**Review by mctavish — APPROVED** [review]

File: docs/SCHEMA.md
Comment (jasonborg) [reply docs/SCHEMA.md:1]: Good call.

File: docs/SCHEMA.md (hunk 1 above)
Comment (jasonborg) [reply docs/SCHEMA.md]: Done.

File: docs/SCHEMA.md (hunk 2 above)
Comment (jasonborg) [reply docs/SCHEMA.md]: Done.

**Pull request merged**